### PR TITLE
CF-852 port drayage and turtlebot sim updates

### PR DESCRIPTION
**Pull request by afrye51**
# PR Details
## Description
Port Drayage leverages Mobility Operation messages contents adhering to the JSON schema defined in https://usdot-carma.atlassian.net/wiki/spaces/CRMFRT/pages/2006482998/Detailed+Design+-+Port+Drayage+Plugin+V2X-Hub#Communication 

The goal of this story is to design and implement basic logic that enables the system to publish a valid Mobility Operation message (that matches the JSON schema linked-to above) upon arrival at its initial manually-set destination. As a reference, CARMA Platform’s construction of an arrival message is handled here: https://github.com/usdot-fhwa-stol/carma-platform/blob/70998021ced231d85fe5dd137906660346bbeb06/port_drayage_plugin/src/port_drayage_worker.cpp#L149

Added turtlebot sim map and params/launch files. Added port drayage nodes to c1t_bringup launch files. Still looking for a way to get relative paths for params files. 

Related PR: https://github.com/usdot-fhwa-stol/navigation2_extensions/pull/3

## Related Jira Key

https://usdot-carma.atlassian.net/browse/CF-852

## How Has This Been Tested?

Tested in VM turtlebot sim with route server running. Confirmed message ingest/generation on cpp_message mobility operation topics. 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: launch/drivers_launch.xml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <launch>
+  <include file="$(find-pkg-share dsrc_driver)/launch/dsrc_driver.py"/>
```
Comment (john-chrosniak) [inline launch/drivers_launch.xml]: Since these are new dependencies could we add `dsrc_driver` and `cpp_message` to the `package.xml`

**Review by john-chrosniak — COMMENTED** [review]
It would be nice if we could have some workaround to avoid having to copy all of the launch files for using the turtlebot. We're also planning to have separate parameter files for the red and blue trucks so I think now would be a good time to figure out how we're going to resolve this.

File: launch/turtlebot_sim_bringup_launch.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<launch>
+  <node pkg="rviz2" exec="rviz2" name="rviz" args="--display-config $(find-pkg-share nav2_bringup)/rviz/nav2_default_view.rviz"/>
```
Comment (willjohnsonk) [inline launch/turtlebot_sim_bringup_launch.xml]: I'm assuming that this rviz config does not include the route server markers. We should probably point to the one that will be in c1t_bringup following this PR: https://github.com/usdot-fhwa-stol/c1t_bringup/pull/16

**Comment by willjohnsonk** [comment]
@john-chrosniak @afrye51 will be good to merge once the README has been updated to reflect new launch arguments

**Comment by john-chrosniak** [comment]
@willjohnsonk @afrye51 Just added documentation on the new launch arguments, let me know what you think

**Pull request merged**